### ISSUE TITLE
Train new UFT models to address user feedback

### ISSUE THREAD

**Issue by 0x000011b** [PygmalionAI/logbooks#1]
Now that we've released and gotten feedback on a series of model sizes (350M, 1.3B, 2.7B and 6B so far), I think training new versions of the models might be a good call - my intuition says that, for the current versions, model performance is still subpar compared to model size.

Off the top of my head, the problems reported so far were:

- Characters repeating sentences or an action across messages too often
- Characters acting embarrassed, stuttering or getting nervous out of nowhere

My plans to tackle these are:

- Incorporating a subset of SODA which can be used with persona annotations into the training dataset
- Doing some data analysis on the current training dataset, looking for things like stuttering, blushing and limiting its % in the final data
- Possibly being more aggressive with the thresholds on the code that attempts to identify and prune "character looping" from the training dataset
- Testing out different training hyperparams, notably lowering LR and increasing/decreasing batch size/gradient accumulation

Open to other suggestions as well.

**Comment by 0x000011b** [comment]
Already started work on this, notes can be found [here](https://github.com/PygmalionAI/logbooks/blob/master/2023-01-11.md).

**Comment by 0x000011b** [comment]
New version released to `master` after a few `dev` versions, considering this done for now.
